recipes/langtest/meta.yaml

```diff
@@ -0,0 +1,85 @@
+{% set name = "langtest" %}
+{% set version = "1.3.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/langtest-{{ version }}.tar.gz
+  sha256: ee1a11d4d7a1447b0f30d53b804ca59748ac910408c4bafd0589c2937ee43b4a
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - python >=3.8,<4.0
+    - poetry-core
+    - pip
+  run:
+    - python >=3.8.1,<4.0
+    - pydantic 1.10.6
+    - transformers 4.28.0
+    - nest-asyncio >=1.5.0,<2.0.0
+    - jsonlines >=3.1.0,<4.0.0
+    - typing-extensions <4.6.0
+    - pandas >=2.0.3,<3.0.0
+    - pyyaml >=6.0.0,<7.0.0
+    - tqdm >=4.65.0,<5.0.0
+    # currently pytorch version 2.0.1 is NOT available on conda-forge
+    - pytorch >=2.0.0,<3.0.0
+  run_constrained:
+    - langchain >=0.0.200,<0.0.201
+    - johnsnowlabs 4.3.5
+    - rouge-score >=0.1.2,<0.2.0
+    - evaluate >=0.4.0,<0.5.0
+    - huggingface_hub >0.16.0
+    - spacy >=3.0.0
+    - openai >0.27.0
+    # - pytorch >=2.0.1,<3.0.0
+    - cohere >=4.10.0,<5.0.0
+    - ai21 >=1.1.0,<2.0.0
+    - metaflow >=2.9.0
+    - accelerate <0.21.0
+    - seqeval >1.2.0
+    - mlflow >=2.5.0,<3.0.0
+    - datasets >=2.14.0
+
+test:
+  imports:
+    - langtest
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://www.langtest.org
+  summary: John Snow Labs provides a library for delivering safe & effective NLP models.
+  license: Apache-2.0
+  license_file: LICENSE
+  license_url: https://github.com/JohnSnowLabs/langtest/blob/main/LICENSE
+  description: |
```

Comment:
Can we trim this description a little? It is quite long and with things that won't render well in the package metadata.

Reply:
Making it shorter. Thank you for the suggestion.